priv/repo/migrations/20230405111657_remove_project_memberships_duplicates.exs

```diff
@@ -0,0 +1,32 @@
+defmodule Ask.Repo.Migrations.RemoveProjectMembershipsDuplicates do
+  use Ecto.Migration
+
+  def up do
+    # Remove duplicates and keep max level in column level for each user and project in project_membnerships
+    execute("""
+      DELETE t1
```

Comment:
Are you sure about `DELETE t1`? Shouldn't it be `DELETE FROM project_memberships t1`?